### ISSUE TITLE
Is it possibe to lock the default browser settings? Missing user based GPO for setting file associations

### ISSUE THREAD

**Issue by crefohb** [MicrosoftDocs/EdgeEnterprise#109]
The results of the above procedure: Users are still able to change the default browser or click "Reset to Microsoft recommended defaults". At next reboot the GPO is re applied and Edge will be default browser again.

Is ist possible to lock the default browser settings by GPO so users can't change them?
Why there is no user based GPO to set default file associations?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2d80a2f5-5b21-a16b-c401-5e1c40be6f00
* Version Independent ID: b63c5b42-443b-687c-a445-58fa4de6b84b
* Content: [Set Microsoft Edge as the default browser on Windows and macOS](https://docs.microsoft.com/en-us/deployedge/edge-default-browser#feedback)
* Content Source: [edgeenterprise/edge-default-browser.md](https://github.com/MicrosoftDocs/EdgeEnterprise/blob/public/edgeenterprise/edge-default-browser.md)
* Product: **microsoft-edge**
* GitHub Login: @dan-wesley
* Microsoft Alias: **brianalt**

**Comment by jbeale53** [comment]
I agree, I'm not sure why it's no longer possible to set this as a user policy.  Makes a lot more sense in our environment.
